modules/oe_content_call_proposals/config/install/core.entity_view_display.node.oe_call_proposals.default.yml

```diff
@@ -87,7 +87,7 @@ content:
     region: content
     label: above
     settings:
-      trim_length: 80
+      trim_length: null
```

Comment:
We can't test maxlength for link title if it is truncated, so let's leave it as is.